pkg/rke2/rke2.go

```diff
@@ -86,6 +80,12 @@ func setup(clx *cli.Context, cfg Config) error {
 		auditPolicyFile = defaultAuditPolicyFile
 	}
 
+	// If system-default-registry is set, retag images imported from tarballs to appear to
+	// come from this registry.
+	if cfg.Images.SystemDefaultRegistry != "" {
```

Comment:
Are there any relevant validations we should be doing on the contents of `cfg.Images.SystemDefaultRegistry`?

Reply:
yeah, `images.NewResolver(cfg.Images)` a couple lines down validates it.

I suppose this could go just above the `defaults.Set` call below but the the effective result is the same if the input is invalid - an error is displayed and rke2 exits.

Reply:
We probably want to do the validation prior to the assignment/storage but not a blocker at this point.